kafka_utils/util/offsets.py

```diff
@@ -546,7 +546,7 @@ def set_consumer_offsets(
 def _nullify_partition_offsets(partition_offsets):
     result = {}
     for partition in partition_offsets:
-        result[partition] = -1
+        result[partition] = 0
```

Comment:
From the pb it looks like it's doing the right thing, but i wasn't able to find any documentation of this behavior. I'm worried that it might change in other versions of Kafka. However, even this behavior will change with a new version, 0 is a valid value and it will eventually run out of retention.